sdks/go/pkg/beam/transforms/sql/sql_test.go

```diff
@@ -0,0 +1,105 @@
+package sql
+
+import (
+	"github.com/apache/beam/sdks/v2/go/pkg/beam"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/transforms/sql/sqlx"
+	"reflect"
+	"runtime"
+	"testing"
+)
+
+func TestOptions_Add(t *testing.T) {
+	o := options{}
+	str := "this is a string"
+	bytes := []byte{1, 2, 3, 4}
+	testOpt := sqlx.Option{
+		Urn:     str,
+		Payload: bytes,
+	}
+	tests := []struct {
+		opt sqlx.Option
+	}{
+		{
+			opt: testOpt,
+		},
+	}
+
+	for _, test := range tests {
+		o.Add(test.opt)
+		if o.customs == nil || !reflect.DeepEqual(o.customs[len(o.customs)-1], test.opt) {
+			t.Errorf("Add failed.")
+		}
+	}
+}
+
+func TestInput(t *testing.T) {
+	name1 := "this is a string"
+	collection1 := beam.PCollection{}
+	tests := []struct {
+		name string
+		in   beam.PCollection
+	}{
+		{
+			name: name1,
+			in:   collection1,
+		},
+	}
```

Comment:
Same commentary here as the previous test case. Likely don't need to use a table test here.